### PR TITLE
SECURITY.rst mistake

### PR DESCRIPTION
**Pull request by dek4nice**

### CODE REVIEW

File: docs/SECURITY.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Filesytem Permissions
+Filesystem Permissions
 =====================
```
Comment (ale-rt) [inline docs/SECURITY.rst]: Thanks a lot!
You should also add a "=" in line 2 to match the length of the sentence in line 1 :)

**Comment by ale-rt** [comment]
Thanks again, @icemac I do not know if @dek4nice has signed the contributor agreement and if it is required for such a small change.

File: docs/SECURITY.rst (hunk 1 above)
Comment (dek4nice) [reply docs/SECURITY.rst]: two commits for two bytes.. overkill :(

Comment (icemac) [reply docs/SECURITY.rst]: No problem, we'll squash merge them into one. 😆

**Comment by icemac** [comment]
@dek4nice Thank you for your pull request. I do not think that it is necessary to sign a contributor agreement for changing those two bytes.

**Comment by tseaver** [comment]
Typo fixes definitely do not meet the bar for "copyrightability."

**Pull request merged**